### PR TITLE
Margin and padding shim for bootstrap 5 migration

### PR DESCRIPTION
**Pull request by hellcp-work**
This is a part of #13602, updating all existing margin and padding to use s and e instead of l and r

### CODE REVIEW

File: src/api/app/assets/stylesheets/webui/bs5-shims.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
```
Comment (dmarcoux) [inline src/api/app/assets/stylesheets/webui/bs5-shims.scss:3]: This is defined [upstream in Bootstrap](https://github.com/twbs/bootstrap/blob/d70b5db2e0aee9ea9a7255d28395278876bf53ba/scss/mixins/_breakpoints.scss#L48-L57).

**Comment by dmarcoux** [comment]
It would be nice to link to the Bootstrap documentation where those changes are mentioned: https://getbootstrap.com/docs/5.0/migration/#utilities

File: src/api/app/assets/stylesheets/webui/bs5-shims.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@each $breakpoint in map-keys($grid-breakpoints) {
```
Comment (dmarcoux) [inline src/api/app/assets/stylesheets/webui/bs5-shims.scss:1]: How did you come up with this Sass code? It looks like something from upstream which was slightly adapted. I'm asking to understand and also to compare with how Bootstrap does this.

**Comment by dmarcoux** [comment]
This PR will need to be rebased to fix conflicts.

**Review by eduardoj — APPROVED** [review]

**Pull request merged**